src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/RestoreOperation/RestoreDatabaseTaskDataObject.cs

```diff
@@ -57,6 +57,8 @@ public interface IRestoreDatabaseTaskDataObject
         string DefaultTargetDbName { get; }
         string TargetDatabaseName { get; set; }
 
+        bool CanDropExistingConnections { get; }
```

Comment:
This property returns if it's possible to drop the connections for a database which is restoring to. 
I use the property in the interface here for testing but basically I use this property to see if user can modify the restore option (CloseConnections)